Compilation fails "use of unsolved module or unlinked crate"

Tried to update hayro to the latest commit in `main`,
now the compilation fails with the following message:

```
error[E0433]: failed to resolve: use of unresolved module or unlinked crate `cpufeatures`
  --> /mnt/Daten/source/rnote/_mesonbuild/cargo-home/registry/src/index.crates.io-1949cf8c6b5b557f/fast_image_resize-5.6.0/src/compat/no_std_impl.rs:13:9
   |
13 |         cpufeatures::new!(cpuid_avx2, "avx2");
   |         ^^^^^^^^^^^ use of unresolved module or unlinked crate `cpufeatures`
```

Searching for `cpufeatures` in my Cargo.lock also is unsuccessful.

it seems to be related to `fast_image_resize` dependency.

It might be the case that if this crate should be used without `std` dependencies,
the explicit [feature `no_std`](https://github.com/Cykooz/fast_image_resize/blob/01889ecd0dc2b0dfee674253cbb9ad5a60377d2c/Cargo.toml#L45) must be enabled.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,7 +33,7 @@ hayro-syntax = { path = "hayro-syntax", version = "0.5.0", default-features = fa
 hayro-svg = { path = "hayro-svg", version = "0.3.0", default-features = false }
 hayro-write = { path = "hayro-write", version = "0.4.0", default-features = false }
 
-bitflags = { version = "2.9" }
+bitflags = { version = "2.10" }
 bytemuck = { version = "1" }
 image = { version = "0.25", default-features = false }
 kurbo = { version = "0.12.0" }
@@ -54,7 +54,7 @@ xmlwriter = { version = "0.1" }
 base64 = { version = "0.22" }
 zune-jpeg = { version = "0.5", default-features = false }
 vello_cpu = { version = "0.0.5", default-features = false, features = ["std", "png", "u8_pipeline"] }
-fast_image_resize = { version = "5", default-features = false }
+fast_image_resize = { version = "5.6.0", default-features = false, features = ["std"] }
 fearless_simd = "0.3.0"
 
 [workspace.lints]

--- a/hayro-demo/Cargo.toml
+++ b/hayro-demo/Cargo.toml
@@ -16,7 +16,7 @@ wasm-bindgen = "0.2"
 js-sys = "0.3"
 console_error_panic_hook = "0.1.7"
 log = "0.4"
-bytemuck = "1.14"
+bytemuck = "1.24"
 hayro = { workspace = true, features = ["embed-fonts"]}
 vello_cpu = { workspace = true }
 

--- a/hayro-jbig2/Cargo.toml
+++ b/hayro-jbig2/Cargo.toml
@@ -22,7 +22,7 @@ workspace = true
 [dev-dependencies]
 image = { workspace = true, default-features = false, features = ["png"] }
 indicatif = { version = "0.17", features = ["rayon"] }
-rayon = "1.10"
+rayon = "1.11"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 

--- a/hayro-jpeg2000/Cargo.toml
+++ b/hayro-jpeg2000/Cargo.toml
@@ -31,7 +31,7 @@ workspace = true
 [dev-dependencies]
 image = { workspace = true, default-features = false, features = ["png"] }
 indicatif = { version = "0.17", features = ["rayon"] }
-rayon = "1.10"
+rayon = "1.11"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 

--- a/hayro-tests/Cargo.toml
+++ b/hayro-tests/Cargo.toml
@@ -27,7 +27,7 @@ pdf-writer = { workspace = true }
 image = { workspace = true, default-features = false}
 resvg = { workspace = true }
 walkdir = "2"
-rayon = "1.8"
+rayon = "1.11"
 memchr = { version = "2.7.6"}
 
 

